cmd/verify_metadata.go

```diff
@@ -144,41 +150,137 @@ func verifyMetadata(cmd *cobra.Command, args []string) error {
         return err
     }
 
+    if metadata_type == "premiuminfo" {
+        return verifyPremiumMetadata(cmd, serviceAccountKeyFilePath, destDir, artifactPath, language, packageID, version, disableCertificateVerification, disableDeletes)
+    }
+
+    return verifyNONPremiumMetadata(cmd, serviceAccountKeyFilePath, destDir, language, packageID, version, metadata_type, disableCertificateVerification, disableDeletes)
```

Comment:
intead of non premium, should be re-name it to standard

Reply:
done